modules/tinymce/src/core/main/ts/api/util/EventDispatcher.ts

```diff
@@ -126,16 +126,16 @@ class EventDispatcher<T> {
   }
 
   /**
-   * Fires the specified event by name.
+   * Dispatches the specified event by name.
    *
-   * @method fire
-   * @param {String} name Name of the event to fire.
+   * @method dispatch
+   * @param {String} name Name of the event to dispatch
    * @param {Object?} args Event arguments.
    * @return {Object} Event args instance passed in.
    * @example
-   * instance.fire('event', {...});
+   * instance.dispatch('event', {...});
    */
-  public fire <K extends string, U extends MappedEvent<T, K>>(name: K, args?: U): EditorEvent<U> {
+  public dispatch <K extends string, U extends MappedEvent<T, K>>(name: K, args?: U): EditorEvent<U> {
```

Comment:
Are we sure we don't want to keep a deprecated fire function around this seems like a pretty massive change to just rename it. Lots of our internal plugins will break and so will likely third party code as well if they use our event system. I'm all for us changing this but we have not had any deprecation messages out for this have we. We don't know how much it's used in the wild etc?

- [ ] Consider adding deprecated fire alternative functions

Reply:
Yeah this was Andy and I mentioned late yesterday afternoon as well. Ben is going to add the `fire` function back and mark it as deprecated. It looks like he moved the Jira back out of review but didn't update this PR (or convert it back to a draft). I'll go ahead and do that now.